roles/satellite-clone/tasks/main.yml

```diff
@@ -60,7 +60,7 @@
   command: subscription-manager repos --enable rhel-7-server-satellite-{{ satellite_version }}-puppet{{ puppet_version }}-rpms
   when:
     - enable_repos
-    - puppet_version | int >= 4
+    - puppet_version in [4]
```

Comment:
Do we know what version of puppet sat 6.4 uses?

Reply:
Puppet 5 is what's on my box

Reply:
So I'm guessing there is no puppet repo for 6.4?

Reply:
Oh, right - it comes from the base sat repo now!